api/run_eval.py

```
@@ -257,9 +262,117 @@ def transcribe_with_retry(
 
                 return "".join(transcript_text) if transcript_text else ""
 
+            elif model_name.startswith("gladia/"):
+
+                def make_request(url, headers, method="GET", data=None, files=None):
+                    if method == "POST":
+                        # For file uploads, use data parameter instead of json
+                        if files:
+                            response = requests.post(
```

Ruff (0.13.2)

Probable use of `requests` call without timeout

(S113)